edge/services/gps/Makefile

```diff
@@ -16,8 +16,8 @@ export TIME_OUT:=20
 
 # Build the docker image for the current architecture
 build:
-	docker build -t $(DOCKER_IMAGE_BASE)_$(ARCH):$(SERVICE_VERSION) -f ./Dockerfile.$(ARCH) .
-
+	docker build --build-arg ARCH=$(ARCH) --platform linux/$(ARCH) -t $(DOCKER_IMAGE_BASE)_$(ARCH):$(SERVICE_VERSION) -f ./Dockerfile . 
+	@docker image prune --filter label=stage=builder --force
```

Comment:
This might be able to be done in a more elegant way, but I removed both these images in the build target after the `prune`:
```
	@docker image rm golang:1.19-alpine --force
	@docker image rm alpine:latest --force
```